templates/docs/shared/_docs.html

```diff
@@ -79,6 +79,9 @@
               Toggle side navigation
             </a>
           </div>
+          <h3 class="p-side-navigation__heading--linked u-no-padding--left" data-level="1">
```

Comment:
I don't like when we add an ad-hoc section for specific pages. I would rather prefer not to have an "Overview" section and be consistent with other pages.